controllers/front/OrderConfirmationController.php

```diff
@@ -79,31 +143,26 @@ public function init()
      */
     public function initContent()
     {
-        if (Configuration::isCatalogMode()) {
```

Comment:
I think this check was on point?

Reply:
I removed this, because imagine a situation - You have catalog mode disabled and ordering enabled. Somebody makes an order, you enable catalog mode 5 minutes after that. Why would he not be able to re-check his order confirm? :-)